behat.yml

```diff
@@ -27,3 +26,10 @@ default:
         login_form_selector: '#user-login-form'
 imports:
   - behat.local.yml
+  - modules/lightning_features/lightning_core/tests/behat.yml
```

Comment:
Is there anything we could do to prevent this from breaking BLT? Or is that something BLT would need to adjust around?